### DISCUSSION TITLE
About the usage of `provisioning-bare-metal.xml`

### DISCUSSION THREAD

**Discussion by liweinan** [resteasy/resteasy-spring#178] [Q&A] [answered]
@jfdenise I'm checking the `glow` setup in tests, and I have a question. What's the usage of the files in the `glow` dir:

- https://github.com/resteasy/resteasy-spring/tree/main/testsuite/layers-metadata-tests/src/test/resources/glow

Are these files mandatory to use the Glow feature?

**Comment by jfdenise** [comment]
@liweinan , yes, that is mandatory for tests. That mimics the official repository: https://github.com/wildfly/wildfly-galleon-feature-packs/ that doesn't contain snapshot versions of resteasy.

**Reply by liweinan** [reply]
With my understanding, this setup will goes into WildFly, so when I'm acting like an end user and deploy the projects into WildFly, the Glow feature will automatically detect the layers I'm using, and I don't have to setup this `glow` dir and its files in my deployment project, am I right?

In addition, are there any references (code, doc, blog posts, etc) that explains the usages and configuration meanings of the `resources/glow` ?

**Comment by jfdenise** [comment, marked as answer]
Yeah, nothing to be done on the user front. That is only for tests.
The README.md in https://github.com/wildfly/wildfly-galleon-feature-packs/ explains what this repository is about.

**Comment by liweinan** [comment]
@jfdenise Thanks for teaching!
